SECURITY.md

```diff
@@ -2,19 +2,13 @@
 
 ## Supported Versions
 
-| Version     | Supported          | Status                | Comments                             |
-| ----------- | ------------------ | --------------------- | ------------------------------------ |
-| 1.8.x       | :white_check_mark: | Long Term Support     | See [Long Term Support policy][0]    |
-| 1.3.x-1.7.x | :x:                |                       |                                      |
-| 1.2.x       | :warning:          | Security patches only | Last version to provide IE 8 support |
-| <1.2.0      | :x:                |                       |                                      |
-
```

Comment:
I think it would be good to leave this table here, and have a date next to the 1.8.x and 1.2.x branches, showing when they ended. Removing this table entirely could leave the person wondering about the exact facts. If you leave this table here with EOL dates, it really drives home the point that it was supported until `12/31/2021`. 

I would also add a piece above the table that says something like:

`[AngularJS is no longer supported](https://goo.gle/angularjs-end-of-life). The following summarizes the End-of-life dates.`